2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2021 @@
+sufficient security for funded software projects, while allowing for
+adaptation of the portfolio of funded software to novel research
+directions and community needs. Failure to meet the reevaluation
+criteria should lead to the decision to phase-out sustainable funding.
+The phase-out process may come with a 1-year funding program with clear
+goals regarding the archiving and preservation of the software,
+documentation, and all existing resources in a consolidation plan.
+
+Who Sustains Research Software?
+===============================
+
+> Kim wants to broaden her research portfolio within researchonomics and
+> applies for postdoctoral positions at other institutions. Her library
+> *hal9k* is growing in popularity within researchonomics, and she wants
+> to continue working on it. As her university has adopted an open
+> science policy, *hal9k* is free software under an open source license,
+> and Kim is free to continue her work on the library even after moving
+> away from UofA.  
+> Due to her involvement in the creation of *hal9k* as well as her
+> previous success in attracting funding, Kim has the choice between
+> multiple, attractive positions and decides to temporarily move to the
+> researchonomics group at Eden University (EdU).She has already
+> extended *hal9k* in multiple directions in the past and plans to
+> continue this work at EdU. Her group leader at EdU would like to
+> continue funding her, but due to a law called the Fixed-term Research
+> Contract Bill, EdU is not allowed to extend her contract, and neither
+> third-party funding for her own position nor a permanent position are
+> available. After having developed a now widely-used research tool,
+> several publications in software and paper form, as well as having
+> attracted funding, Kim finds herself looking for a job again.
+
+Research relies on software and software relies on the people developing
+and maintaining it. Sustainable research requires sustainable software,
+and this in terms requires continuity for those who develop and maintain
+it.
+
+Requirements
+------------
+
+Possibly the most important demand is the need for an increase in
+**recognition and awareness** of research software as a first class
+citizen in research (Akhmerov, Cruz, Drost, Hof, Knapen, Kuzak,
+Martinez-Ortiz, et al. 2019; Chue Hong 2019; Scheliga et al. 2019). For
+sustainability of research software, long-term commitments of the
+respective software leads are crucial, but no **professional RSE
+profiles** currently exist. In consequence, it is essential to create
+career paths for research software engineers (RSEs) that are attractive
+and include permanency. While creating permanent positions in the German
+academic system below the faculty level is an actively discussed topic
+overall (Vereinigung der Kanzlerinnen und Kanzler der Universitäten
+Deutschlands 2019), we specifically focus on the needs stemming from the
+development and maintenance of research software here.
```

Comment:
this whole section is very university centric. How about Helmholtz' HiFiS? Was bei Leibniz Open Science abgeht kann ich nicht einschätzen

Reply:
Are there citable works from non-uni institutions regarding this?